src/main/java/com/redhat/platform/experience/TypescriptAxiosWebpackModuleFederationGenerator.java

```diff
@@ -251,7 +251,6 @@ protected void updatePropertyForAnyType(CodegenProperty property, Schema p) {
   @Override
   public OperationsMap postProcessOperationsWithModels(OperationsMap objs, List<ModelMap> allModels) {
       objs = super.postProcessOperationsWithModels(objs, allModels);
-      this.updateOperationParameterForEnum(objs);
```

Comment:
@Hyperkid123 Do you know anything about this function? Before removing it, I was getting duplicate string names in `api.ts` in enums, e.g. it would use `ListPermissionListPermissionOptionsFieldEnum` instead of `ListPermissionOptionsFieldEnum` in the comments and in some `index.ts` endpoint files which would cause typing issues.

Reply:
I indeed don't know what it does/did. I'll dig into it a bit.